src/modules/Exchange.ts

```diff
@@ -587,4 +528,34 @@ export class Exchange {
     );
     return uint256ToHumanTokenAmount(allowance, decimals);
   }
+
+  encodeZeroExExchangeData(
+    proxyExchangeData: {
+      tokenFrom: string,
+      allowanceTarget: string,
+      minUsdcAmount: string,
```

Comment:
Prefer BigNumberable here (consistent with methods above) if necessary, convert to string before passing to web3